src/styles/block/stopwatch.scss

```diff
@@ -0,0 +1,66 @@
+.stopwatch {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 80vmin;
+  height: 80vmin;
+  line-height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  margin: 69px auto;
+
+  &__minute-hand {
+    position: absolute;
+    bottom: 50%;
+    left: 48%;
+    transform-origin: bottom center;
+    transform: rotate(0deg);
+    width: 3vmin;
+    height: 20vmin;
+    background-color: #0700ff;
+
+    animation: rotate-minute 600s steps(60) infinite; /* Минутная стрелка за час */
```

Comment:
The animation duration for the minute hand seems to be off. According to a real-world clock, the minute hand makes a full rotation in 3600 seconds (60 minutes). The provided duration of 600 seconds will make the minute hand rotate six times faster than it should.